### PR TITLE
Update and re-enable UCS tests

### PR DESCRIPTION
**Pull request by geoff-reid**
@RackHD/corecommitters @RackHD/veyron

### CODE REVIEW

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by srinia6** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

File: test/config/stack_config.json [hunk 1]
```diff
@@ -12,11 +12,23 @@
...
     "ucsm_pass" : "ucspe",
     "ucs_service_uri" : "https://localhost:7080",
     "ucsm_config_file" : "./tests/ucs/ucsm_config.xml"
   },
+  "hop_stack":{
+    "rackhd_host": "localhost",
+    "type": "vagrant",
+    "ucsm_ip" : "10.240.19.70",
+    "ucsm_user" : "admin",
+    "ucsm_pass" : "Zilog256",
```
Comment (brianparry) [inline test/config/stack_config.json]: @geoff-reid We should probably redact this password somehow.  Perhaps there is a way to pass it in with an environment variable?

File: test/config/stack_config.json [hunk 2]
```diff
@@ -12,11 +12,17 @@
...
-    "ucsm_user" : "ucspe",
-    "ucsm_pass" : "ucspe",
+    "ucsm_ip" : "172.31.128.252",
     "ucs_service_uri" : "https://localhost:7080",
     "ucsm_config_file" : "./tests/ucs/ucsm_config.xml"
   },
+  "hop_stack":{
+    "rackhd_host": "localhost",
+    "type": "vagrant",
+    "ucsm_ip" : "10.240.19.70",
```
Comment (panpan0000) [inline test/config/stack_config.json]: please don't use a EMC internal IP into github

**Pull request merged**